script.sh

```
@@ -0,0 +1,25 @@
+    update_tags["$tag_pointer"]+="$tag;"
+  fi
+done
+
+for tag in ${!update_tags[@]}; do
+  git checkout $tag
+
+  declare -a new_tags
+  IFS=$';' read -ra new_tags <<< "${update_tags[$tag]}"
+  for new_tag in ${new_tags[@]}; do
+    echo "${new_tag}"
+    git tag -a "${new_tag}" -m "Update tag ${new_tag} to point to ${tag}"
```

Shellcheck (0.10.0)

[error] Couldn't parse this double quoted string. Fix to allow more checks.

(SC1073)